packages/terra-consumer-nav/src/icons/IconInfo.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
+import React from 'react';
+import IconBase from 'terra-icon/lib/IconBase';
+
+const SvgIcon = (customProps) => {
+  const attributes = Object.assign({}, customProps);
```

Comment:
specific reason for the object assign as opposed to just deconstructing and spreading customProps like in other elements?